.travis.yml

```diff
@@ -73,7 +74,7 @@ _macosx_nodejs_template: &_macosx_nodejs_template
   rust: nightly
 
 _to_deploy_template: &_to_deploy_template
-  if: tag =~ /^v\d+\.\d+\.\d+/
+  if: tag =~ /^holochain-nodejs\d+\.\d+\.\d+/
```

Comment:
Might be nice to have a dash if possible or even a dash-vee: 
`holochain-nodejs-1.2.3` or
`holochain-nodejs-v1.2.3`

Reply:
you know, I like that idea. Lets do it